gtest/src/log.rs

```diff
@@ -21,8 +23,34 @@ impl CoreLog {
             dest: other.dest,
             payload: other.payload.into_raw(),
             exit_code: other.reply.map(|(_, code)| Some(code)).unwrap_or_default(),
+            id: other.id,
         }
     }
+
+    pub(crate) fn generate_reply(
+        &self,
+        payload: Payload,
+        message_id: MessageId,
+        value: u128,
+    ) -> Message {
+        Message {
+            source: self.dest,
+            dest: self.source,
+            payload,
+            gas_limit: None,
+            value,
+            id: message_id,
+            reply: self.exit_code.map(|exit_code| (self.id, exit_code)),
+        }
+    }
+
+    pub fn get_payload(&self) -> Payload {
+        self.payload.clone().into()
+    }
+
+    pub fn get_id(&self) -> MessageId {
```

Comment:
I'd name getters shorter.
```suggestion
    pub fn id(&self) -> MessageId {
```